### DISCUSSION TITLE
Why is the search box not working?

### DISCUSSION THREAD

**Discussion by dongrentianyu** [web-infra-dev/rspress#501] [Q&A] [answered]
是需要做一些什么操作吗？本地和github部署的都没有效果。

**Comment by dongrentianyu** [comment]
![image](https://github.com/web-infra-dev/rspress/assets/109711900/8da004e4-600b-4aea-8d64-66f8491af70c)

就这样一直转。

**Comment by dongrentianyu** [comment]
还是说需要安装插件？

**Reply by sanyuan0704** [reply]
抱歉，这是新版本的一个 bug，我来修复一下，等下发个版给你

**Reply by sanyuan0704** [reply, marked as answer]
rspress 1.9.1 版本已经修复了哈

**Reply by dongrentianyu** [reply]
那rspress怎么更新呢？在文档里好像没看到明确的命令用来升级。

---

用`pnpm update`更新到1.9.1版本了。但好像还是没有实现搜索功能。

是还需要写什么代码来导入吗？

---

还是不知道咋处理。

![image](https://github.com/web-infra-dev/rspress/assets/109711900/24d5d1c8-8ba4-4a5d-acc5-067074d71512)

没有报错。但搜索框就是一直转。跟上面一样。

**Reply by sanyuan0704** [reply]
你这里的标题 2021 应该加引号，声明为字符串，否则搜索逻辑中会报错：

![image](https://github.com/web-infra-dev/rspress/assets/39261479/173c5f85-ef93-4b36-8883-a91dfcc7787a)

Rspress 内部在下一个版本会自动转字符串哈。

**Reply by sanyuan0704** [reply]
@dongrentianyu

**Reply by dongrentianyu** [reply]
感谢指导。现在已经恢复了搜索功能。就是只能在搜索后显示标题，不能像文档里那样，把正文里的内容也展示出来。希望文档能更详细补充一下，如何用内置组件实现正文内容搜索。还是说我在mdx里用了`<div>`标签包裹而无法这样做。

再次感谢指导，祝您节日愉快。

**Reply by sanyuan0704** [reply]
mdx 标签里面包裹的内容不会被搜到的哈

**Reply by dongrentianyu** [reply]
好的，了解了。
